app/templates/components/public/side-menu.hbs

```diff
@@ -3,32 +3,44 @@
     {{#scroll-to href='#info' class='item active'}}
       {{t 'Info'}}
     {{/scroll-to}}
-    {{#scroll-to href='#tickets' class='item'}}
-      {{t 'Tickets'}}
-    {{/scroll-to}}
-    {{#scroll-to href='#speakers' class='item'}}
-      {{t 'Speakers'}}
-    {{/scroll-to}}
+    {{#if event.tickets}}
+      {{#scroll-to href='#tickets' class='item'}}
+        {{t 'Tickets'}}
+      {{/scroll-to}}
+    {{/if}}
+    {{#if event.speakers}}
+      {{#scroll-to href='#speakers' class='item'}}
+        {{t 'Speakers'}}
+      {{/scroll-to}}
+    {{/if}}
   {{else}}
-    <a class="item" href="{{href-to 'public.index'}}">
-      {{t 'Info'}}
-    </a>
+  <a class="item" href="{{href-to 'public.index'}}">
+    {{t 'Info'}}
+  </a>
+  {{#if event.isTicketingEnabled}}
     <a class="item" href="{{href-to 'public.index'}}#tickets">
       {{t 'Tickets'}}
     </a>
+  {{/if}}
+  {{#if event.speakers}}
     <a class="item" href="{{href-to 'public.index'}}#speakers">
       {{t 'Speakers'}}
     </a>
   {{/if}}
-  {{#link-to 'public.sessions' class='item'}}
-    {{t 'Sessions'}}
-  {{/link-to}}
+  {{/if}}
+  {{#if event.isSessionsSpeakersEnabled}}
+    {{#link-to 'public.sessions' class='item'}}
+      {{t 'Sessions'}}
+    {{/link-to}}
+  {{/if}}
   {{#link-to 'public.schedule' class='item'}}
     {{t 'Schedule'}}
   {{/link-to}}
-  {{#link-to 'public.cfs' class='item'}}
-    {{t 'Call for Speakers'}}
-  {{/link-to}}
+  {{#if event.speakersCall}}
```

Comment:
This will cause a network request to speakers call. Use `is-sessions-speakers-enabled ` from events response for this.